packages/components/src/range-control/index.tsx

```diff
@@ -41,6 +41,15 @@ import { space } from '../utils/space';
 
 const noop = () => {};
 
+function useUniqueId( idProp?: string ) {
+	const id = useInstanceId(
+		UnforwardedRangeControl,
+		'inspector-range-control'
+	);
+
+	return idProp || id;
```

Comment:
I had that same question when reviewing the original PR, and at the time we agreed that `||` was a good choice because empty strings would not make a good `id`. https://github.com/WordPress/gutenberg/pull/48498#discussion_r1122433330

Reply:
I forgot about it — thank you for pointing it out!